app/models/agency.rb

```diff
@@ -0,0 +1,3 @@
+class Agency < ApplicationRecord
+  validates :name, presence: true
```

Comment:
Can we add the proper `has_*` relationship with agency identity, such as `has_many :agency_identities`?